specs/online_order_spec.rb

```diff
@@ -55,17 +73,41 @@
       # TODO: Your test code here!
       # Useful checks might include:
       #   - OnlineOrder.all returns an array
+      it "Returns an array object" do
+        online_orders = Grocery::OnlineOrder.all
+        online_orders.must_be_kind_of Array #or Array < Object
+      end
       #   - Everything in the array is an Order
+      it "Everything in the array is an Order" do
+        online_orders = Grocery::OnlineOrder.all
+        online_orders.must_be_kind_of Grocery::Order
+      end
       #   - The number of orders is correct
+      it "Returns the correct number of orders" do
+        online_orders = Grocery::OnlineOrder.all
+        online_orders.length.must_equal 100
+      end
       #   - The customer is present
+      it "Returns an object for the customer" do
+        online_orders = Grocery::OnlineOrder.all
+        online_orders[1].customer.wont_be_nil
+      end
+
       #   - The status is present
-      # Feel free to split this into multiple tests if needed
+      it "Returns a value for the order status" do
+      online_orders = Grocery::OnlineOrder.all
+      online_orders[1].order_status.wont_be_nil
+      end
+
     end
   end
-  
+
   describe "OnlineOrder.find_by_customer" do
     it "Returns an array of online orders for a specific customer ID" do
-      # TODO: Your test code here!
+      online_orders = Grocery::OnlineOrder.all
+      online_orders.find("100").must_be_kind_of Array
```

Comment:
you should be testing `Grocery::OnlineOrder.find_by_customer()` in this test, not `.find` on an array of all Online Orders